test/toml_test.rb

```diff
@@ -84,4 +85,34 @@ def test_line_break
     parsed = TOML.parse("hello = 'world'\r\nline_break = true")
     assert_equal({ 'hello' => 'world', 'line_break' => true }, parsed)
   end
+
+  def compare_toml_files(folder, file = nil, &block)
```

Comment:
This is a great method, only used by `test_valid_cases` but not for `test_invalid_cases`. Is there any reason to do that?

Also, this should be a `private` I guess.